tools/getrealdeps.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+# getrealdeps for Chromebrew
+# Author: Satadru Pramanik (satmandu) satadru at gmail dot com
+# set -x
+pkg="${1}"
+
+RED='\e[1;91m';    # Use Light Red for errors.
+YELLOW='\e[1;33m'; # Use Yellow for informational messages.
+GREEN='\e[1;32m';  # Use Green for success messages.
+BLUE='\e[1;34m';   # Use Blue for intrafunction messages.
+GRAY='\e[0;37m';   # Use Gray for program output.
+MAGENTA='\e[1;35m';
+RESET='\e[0m'
+
+# Simplify colors and print errors to stderr (2).
+echo_error() { echo -e "${RED}${*}${RESET}" >&2; }
+echo_info() { echo -e "${YELLOW}${*}${RESET}" >&1; }
+echo_success() { echo -e "${GREEN}${*}${RESET}" >&1; }
+echo_intra() { echo -e "${BLUE}${*}${RESET}" >&1; }
+echo_out() { echo -e "${GRAY}${*}${RESET}" >&1; }
+echo_other() { echo -e "${MAGENTA}${*}${RESET}" >&2; }
+
+# Install grep if a functional local copy does not exist.
+if grep --version &> /dev/null; then
+  GREP=grep
+else
+  crew install grep
+  GREP=/usr/local/bin/grep
+fi
+
+: "${CREW_LIB_PREFIX:=$(crew const | $GREP CREW_LIB_PREFIX | awk -F = '{print $2}')}"
```

Comment:
We need to install `awk` also since it is not available out-of-box

Reply:
Awk comes from the mawk package, which is in core.